akd/src/proof_structs.rs

```diff
@@ -119,14 +120,33 @@ impl<H: Hasher> Clone for NonMembershipProof<H> {
 )]
 #[cfg_attr(feature = "serde_serialization", serde(bound = ""))]
 pub struct AppendOnlyProof<H: Hasher> {
+    /// Proof for a single epoch being append-only
+    pub proofs: Vec<SingleAppendOnlyProof<H>>,
+    /// Epochs over which this audit is being performed
+    pub epochs: Vec<u64>,
```

Comment:
nit: These need to be a contiguous range right? Maybe it's better to have start + end epochs, since we don't need all the numbers in the middle